gitlab-ci.yml

```diff
@@ -0,0 +1,42 @@
+cache:
+  key:
+    files:
+      # Cache npm packages using lock file
+      - package-lock.json
+  paths:
+    - .npm/
+
+before_script:
+    - npm ci --cache .npm --legacy-peer-deps --loglevel error
```

Comment:
As a library, we need to keep it up to date  so, it won't be wise to add `--legacy-peer-deps` and `--loglevel error.
Let's remove it and make sure, it will run over CI too.